odh-operator-image/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG version=v0.8.0
+FROM quay.io/opendatahub/opendatahub-operator:${version}
+ARG version
+
+USER root
+RUN microdnf install -y wget && microdnf clean all && rm -rf /var/cache/yum
```

Comment:
Can we avoid this by using `curl -L -O`?

Reply:
Done